### DISCUSSION TITLE
Sync feature description

### DISCUSSION THREAD

**Discussion by xmgz** [spacecowboy/Feeder#201] [Q&A] [answered]
Please would you provide any background on **how sync feature works**. Thank you.

* server sustained by donations? 
* Privacy Policy, how data is (or is not) stored on the server
* do you use this data for statistic analisys? do you share it? do you sell it?
* is data anonimized?
* are user data just synced so it allways remains in user's devices? or are stored so it remais as a backup?
* are E2E encrypted?

Some questions seem obvious, but I think they are legit :wink: in this context. 

thank you

**Comment by spacecowboy** [comment, marked as answer]
>     * server sustained by donations?

yes. See ko-fi page.

>     * Privacy Policy, how data is (or is not) stored on the server

The policy is I don't want your data so I keep none of it

>     * do you use this data for statistic analisys? do you share it? do you sell it?

There is no data to share really

>     * is data anonimized?

In the sense that no personal data is saved

>     * are user data just synced so it allways remains in user's devices? or are stored so it remais as a backup?

Stuff is stored server side and deleted if left unused for too long
 
>     * are E2E encrypted?
 
yes, which is why there is no data to sell

> Some questions seem obvious, but I think they are legit 😉 in this context.

indeed.

to clarify, on "registration" a random accountid is generated together with a random encryption key.

server sees only accountid and encrypted data
it can see how many devices are tied to an account, but not even the device names are unencrypted.

there are ip addresses in the webserver log but they are never explicitly saved anywhere.

**Reply by spacecowboy** [reply]
and encryption key is never sent to the server

**Comment by Lyndeno** [comment]
Is there a way to sync to a desktop app of some kind?

**Reply by spacecowboy** [reply]
No.

**Reply by spacecowboy** [reply]
You can export an OPML and transfer that, but there is no recurring synchronization possible. 

Feeder was built explicitly to not have that.
